core/src/main/java/io/atomix/core/Atomix.java

```diff
@@ -286,6 +286,7 @@ public String toString() {
    */
   public static class Builder implements io.atomix.utils.Builder<Atomix> {
     protected static final String DEFAULT_CLUSTER_NAME = "atomix";
+    protected static final int DEFAULT_COORDINATION_PARTITIONS = 7;
```

Comment:
Why 7 and 71?

Reply:
7 nodes is typical of large consensus based clusters and allows a leader per node, and 71 is a not-too-small, not-too-large prime number

Reply:
code comment explaining the magic numbers?